docs/Manifest.toml

```diff
@@ -1,31 +1,33 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.2"
+julia_version = "1.8.0-rc3"
```

Comment:
This is possibly not a good idea.

Reply:
(In case the manifest format has changed between 1.7 & 1.8. Although I don't think it has)

Reply:
Agreed. I'm not entirely sure that we need this PR anyway though